requirements-dev.txt

```diff
@@ -13,3 +13,6 @@ sphinx
 
 # For `make package`
 wheel
+
+# For datetime tests
+isodate
```

Comment:
Along those lines, is there any special reason why isodate was chosen? It seems like a decent library, with roughly 2,800 downloads per day and decent compatibility with various Python versions. However, there are other more popular libraries which target the same problem area. For example, python-dateutil has roughly 232,000 downloads per day, is compatible with Python 3.5, and has been around for ~6 years longer.

Even more fundamentally, do we need a new dependency at all? Does Pulp state whether it complies with ISO 8601, or the more liberal RFC 3339, or something else?